src/Administration/Resources/app/administration/src/app/init-post/amplitude.init.spec.js

```diff
@@ -139,4 +162,107 @@ describe('src/app/post-init/amplitude.init.ts', () => {
             expect(track).toHaveBeenCalledWith(trackedData.eventName, trackedData.properties);
         });
     });
+
+    describe('user identification', () => {
+        const testShopId = 'knneBsx7LiKySnUq';
+        const testUserId = '8b8ebef4-7fa3-4844-ab7e-120463ea558b';
+        let originalShopId = null;
+
+        beforeEach(() => {
+            jest.clearAllMocks();
+
+            originalShopId = Shopware.Store.get('context').app.config.shopId;
+            Shopware.Store.get('context').app.config.shopId = testShopId;
+        });
+
+        afterEach(() => {
+            if (originalShopId !== null) {
+                Shopware.Store.get('context').app.config.shopId = originalShopId;
+                originalShopId = null;
+            }
+        });
```

Comment:
I think it is not necessary to reset the shop ID here. Tests run in separate contexts for each file.